### PR TITLE
Improve log-in analytics

### PR DESCRIPTION
**Pull request by xsyann**
Rename `Sign In` event to `Log In` and throws it even when the user resume log-in.

The property `type` (`password` or `resume`) allows to track both types of log-in.

### CODE REVIEW

**Review by Donorhan — APPROVED** [review]
Thanks, it's ok for me! I'm waiting for a review from @RobinLbt

**Comment by RobinLbt** [comment]
Hey @xsyann, why do you want to rename the event?

It's a bit annoying on our side since we'll have to join the old event `Sign In` with the new `Log In` every time we want to make an analysis with all the historical data we already tracked

As for the `type` property, it makes sense but most analytics tools have default session events

For instance, we use Mixpanel and we have those: 
image

Which analytics tool are you using?
Maybe they have it too. 
If not, then yes we can add the property :)

**Comment by xsyann** [comment]
Hi @RobinLbt,

I just wanted to rename the event for two reasons:

1. When I read `Sign In`, I think more about a form input signing in, and as I added the resume login event, I thought that `Log In` was more accurate than `Sign In`
2. To keep a consistency with the `Log out` event that I wanted to add

Nonetheless, if it induces a complexity on your side, I'm perfectly fine with 'Sign In', it doesn't really matter :)

We are also using Mixpanel, I guess you are using timeout based session then?
I thought that it was not very expensive to configure event based session if we added exhaustive login events (password + resume) and logout events:

Screenshot 2022-08-29 at 08 37 02

**Comment by RobinLbt** [comment]
Yes you are right, we are using a timeout-based session which is not ideal to be honest 

The event-based solution seems better

I do not see the new `Log Out` event in your commit, is it intentional?

**Comment by xsyann** [comment]
I've done it in a separated branch, I've just created the PR: https://github.com/l3mpire/lemverse/pull/85

**Comment by Donorhan** [comment]
@RobinLbt Is everything good for you?

**Pull request merged**
